.github/workflows/create-apt-repo.yml

```diff
@@ -16,13 +16,11 @@ jobs:
       fail-fast: false
       matrix:
         category:
-          # TODO: replace refs with real ones once PR is approved
           - name: stable
-            # ref: v1.4.0
-            ref: stable_deb_testing
+            ref: 1.3.0  # Previous stable branch + backport
```

Comment:
this should really be 1.3.3

Reply:
So backporting on top of the v1.3.3 tag is way more annoying but 1.3.0 is a branch that currently sits at 1.3.3, so it sort of works. Future stables would indeed use a tag.

Reply:
Oh, joy. Fair. 😄